package.json

```diff
@@ -109,20 +109,22 @@
     "webpack-notifier": "1.3.0"
   },
   "scripts": {
+    "check-env": "sh ./scripts/validate-engine-versions",
     "clean": "rm -rf dist",
     "shrinkwrap": "npm shrinkwrap --dev && ./node_modules/.bin/gulp fixShrinkwrap",
+    "preinstall": "npm run check-env",
     "prebuild-assets": "./node_modules/.bin/gulp checkDependencies && npm install compression-webpack-plugin@latest",
     "build-assets": "npm run clean && NODE_ENV=production webpack --config ./webpack/webpack.production.babel.js",
     "postbuild-assets": "rm -rf node_modules/compression-webpack-plugin",
     "build": "npm run lint && npm test && npm run build-assets",
     "build-with-external-plugins": "npm run lint && npm run lint-plugins && npm test && npm run build-assets",
     "lint": "bash -c 'opts=$@ ; if [[ $# -lt 1 ]]; then opts=\"src/\\*\\*/\\*.js\"; fi ; ./node_modules/.bin/eslint --quiet ${opts}' 0",
     "lint-plugins": "PLUGINS=$npm_config_externalplugins; npm run lint -- \"$PLUGINS/**/*.js\"",
-    "prebuild": "./scripts/validate-tests",
+    "prebuild": "sh ./scripts/validate-tests",
```

Comment:
I've been burnt by this in the passed, where it would run bash on mac, but shell in linux environments and fail. Just wanted to make sure that it was working for the environments we were using.
Do you have a suggestion to make it better?